src/components/Content/index.tsx

```diff
@@ -65,38 +63,69 @@ export const Content: React.VFC = () => {
   return (
     <PageGrid>
       <Header />
-      <Switch>
-        <Route path={routes.task} component={Task} />
-        <Route path={routes.configurePatch} component={ConfigurePatch} />
-        <Route exact path={routes.patch} component={VersionPage} />
-        <Route path={routes.version} component={VersionPage} />
-        <Route path={routes.jobLogs} component={JobLogs} />
-        <Route path={routes.hosts} component={Hosts} />
-        <Route path={routes.host} component={Host} />
-        <Route path={routes.myPatches} component={MyPatches} />
+      <Routes>
+        <Route path={baseRoute.task}>
+          <Route path={tab} element={<Task />} />
+          <Route path="" element={<Task />} />
+        </Route>
```

Comment:
For these tabbed routes does `${route.someRoute}*` work?
This way we an avoid the fallbacks for when the tab isn't added.